roaring/roaring_internal_test.go

```diff
@@ -3225,3 +3227,37 @@ func runContainerFunc(f interface{}, c ...*Container) *Container {
 	}
 	return nil
 }
+
+func TestUnmarshalStdRoaring(t *testing.T) {
```

Comment:
let's expand the word `Standard` here (or maybe even refer to it as "Official"?)